### PR TITLE
[PT-BR] DOM Manipulation.md file translation in the docs section

### PR DESCRIPTION
**Pull request by davidalves1**

### CODE REVIEW

**Comment by khaosdoctor** [comment]
Hey @davidalves1, is this PR ready? It's defined as WIP.

If it is not, can you please convert it to draft?

**Comment by khaosdoctor** [comment]
Hey @davidalves1 are you still with us on this PR?

**Comment by davidalves1** [comment]
Hello @khaosdoctor, I had to pause the translation for a few days, but I already have 70% ready. I think I can finish this weekend

**Comment by khaosdoctor** [comment]
Awesome! Thanks a lot!

**Comment by davidalves1** [comment]
Hey @khaosdoctor, now is able to merge!

File: packages/documentation/copy/pt/tutorials/DOM Manipulation.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+translatable: true
+---
+
+## Manipulação do DOM
+
+### _Uma exploração no tipo `HTMLElement`_
+
+Nos mais de 20 anos desde sua padronização, o JavaScript tem percorrido um longo caminho. Enquanto em 2020, o JavaScript pode ser usado em servidores, em ciência de dados, e até mesmo em dispositivos Internet das Coisas (IoT), é importante lembrar seu caso de uso mais popular: navegadores web.
+
+Websites são feitos de documentos HTML e/ou XML. Estes documentos são estáticos, eles não mudam. O *Modelo de Objeto de Documento (DOM)* é uma interface de programação implementado por navegadores para tornar sites estáticos funcionais. A API do DOM pode ser usada para alterar a estrutura do documento, estilo e conteúdo. A API é tão poderosa que inúmeras estruturas de front-end (jQuery, React, Angular, etc.) foram desenvolvidos em torno dele, a fim de tornar os sites dinâmicos ainda mais fáceis de desenvolver.
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/DOM Manipulation.md]:
```suggestion
Websites são feitos de documentos HTML e/ou XML. Estes documentos são estáticos, eles não mudam. O *Modelo de Objeto de Documento (DOM)* é uma interface de programação implementada por navegadores para tornar sites estáticos funcionais. A API do DOM pode ser usada para alterar a estrutura do documento, estilo e conteúdo. A API é tão poderosa que inúmeras ferramentas de front-end (jQuery, React, Angular, etc.) foram desenvolvidos em torno dele, a fim de tornar os sites dinâmicos ainda mais fáceis de desenvolver.
```

File: packages/documentation/copy/pt/tutorials/DOM Manipulation.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+    <html lang="en">
+      <head><title>Manipulação do DOM com TypeScript</title></head>
+      <body>
+        <div id="app"></div>
+        <!-- Assumindo que index.js é a saída compilada de index.ts -->
+        <script src="index.js"></script>
+      </body>
+    </html>
+
+Lets explore a TypeScript script that adds a `<p>Olá, Mundo!</p>` element to the `#app` element.
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/DOM Manipulation.md]:
```suggestion
Lets explore a TypeScript script that adds a `<p>Olá, Mundo!</p>` element to the `#app` element.
```

I think you missed a spot here

File: packages/documentation/copy/pt/tutorials/DOM Manipulation.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+## A interface `Node`
+
+A função `document.getElementById` retorna um `HTMLElement`. A interface `HTMLElement` extende a interface `Element` que, por sua vez, extende a interface `Node`. Essa extenção a nível de protótipo permite a todos `HTMLElements` a utilizar um subconjunto de métodos padrão. No trecho de código, nós usamos uma propriedade definida na interface `Node` para anexar o novo elemento `p` ao website.
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/DOM Manipulation.md]:
```suggestion
A função `document.getElementById` retorna um `HTMLElement`. A interface `HTMLElement` extende a interface `Element` que, por sua vez, extende a interface `Node`. Essa extensão a nível de protótipo permite a todos `HTMLElements` a utilizar um subconjunto de métodos padrão. No trecho de código, nós usamos uma propriedade definida na interface `Node` para anexar o novo elemento `p` ao website.
```

File: packages/documentation/copy/pt/tutorials/DOM Manipulation.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+### `Node.appendChild`
+
+A última linha do trecho de código é `app?.appendChild(p)`. A seção anterior, `document.getElementById`, detalha o que o operador _optional chaining_  é usado aqui porquê `app` pode ser potencialmente nulo durante a execução. O método `appendChild` é definido por:
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/DOM Manipulation.md]:
```suggestion
A última linha do trecho de código é `app?.appendChild(p)`. A seção anterior, `document.getElementById`, detalha o que o operador _optional chaining_  é usado aqui porque `app` pode ser potencialmente nulo durante a execução. O método `appendChild` é definido por:
```

File: packages/documentation/copy/pt/tutorials/DOM Manipulation.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+
+```ts
+appendChild<T extends Node>(newChild: T): T;
+```
+
+Este método funciona de forma semelhante ao método `createElement` com o parâmetro genérico `T` sendo inferido do argumento `newChild`. `T` é _restrito_ a outra interface base `Node`. 
+
+## Diferença entre `children` e `childNodes`
+
+Anteriormente, este documento detalhou a interface `HTMLElement` extendendo de `Element` que extende de `Node`. Na API DOM existe um conceito de elementos _filhos_. Por exemplo no HTML seguinte, as tags `p` são filhas do elemento `div`
```
Comment (khaosdoctor) [inline packages/documentation/copy/pt/tutorials/DOM Manipulation.md]:
```suggestion
Anteriormente, este documento detalhou a interface `HTMLElement` extendendo de `Element` que estende de `Node`. Na API DOM existe um conceito de elementos _filhos_. Por exemplo no HTML seguinte, as tags `p` são filhas do elemento `div`
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Some changes, there's a place you missed translation too, I think

**Comment by davidalves1** [comment]
@khaosdoctor all done!

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**